src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -276,6 +276,7 @@ module CommandLineArguments =
 /// contains specific names used within Q# dlls
 module DotnetCoreDll = 
     let ResourceName = "__qsharp_data__.bson"
```

Comment:
Maybe we should deprecate this one, too, though with the same comment that it is still in use for older dlls.

Reply:
Deprecated with an appropriate message.